client/store/authStore.ts

```diff
@@ -31,7 +31,7 @@ export const signUp = createAsyncThunk(
         email,
         password,
       });
-      return user;
+      // return user;
```

Comment:
clean up, we need to do one or the other but not call mutate twice (not your fault it was weird how it was set up)